types/lodash/ts3.1/fp.d.ts

```diff
@@ -4020,10 +4020,10 @@ declare namespace _ {
     type LodashSortedUniqBy1x2<T> = (iteratee: lodash.ValueIteratee<T>) => T[];
     interface LodashSplit {
         (separator: RegExp | string): LodashSplit1x1;
-        (separator: lodash.__, string: string): LodashSplit1x2;
-        (separator: RegExp | string, string: string): string[];
+        (separator: lodash.__, string: string | null): LodashSplit1x2;
```

Comment:
Should this include undefined?
If you generated this with the script, you may need to change the source type from `string?: string|null` to `string: string|null|undefined`.
I think that's better anyway because there isn't really a use case for calling split() with no parameters.

Reply:
Makes sense. `_.split()` _technically_ works, but it's not a useful thing to do. I'll make the change.

Reply:
FIxed.